src/microbe_stage/components/MicrobeControl.cs

```diff
@@ -69,6 +69,11 @@ public struct MicrobeControl
     /// </summary>
     public bool Sprinting;
 
+    /// <summary>
+    ///   Whether this microbe is can sprint
+    /// </summary>
+    public bool CanSprint = true;
```

Comment:
Components should be in a fine state when default struct constructed, which sets everything to its type specific default value. So this having true as the default value is not good. Instead this bool should be the reverse (see for example the various `StateApplied` flags that exist instead of there being `Dirty` flags). So this should be something like `IsSprintingExhausted` which would be false by default and true when too exhausted to be able to sprint.